keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -383,6 +399,8 @@ Describe the metrics themselves and the reasons why they weren't added (e.g., co
 implementation difficulties, etc.).
 -->
 
+No.
```

Comment:
Yes, as described in the `How can an operator determine if the feature is in use by workloads?` question